examples/sample-foundry-project/cannonfile.consumer.toml

```diff
@@ -3,18 +3,18 @@ version = "0.0.1"
 description = "Simple project to verify the functionality of cannon"
 keywords = ["sample", "consumer"]
 
-[setting.change_greeting2]
-defaultValue = "new greeting!"
+[var.change_greeting2]
+value = "new greeting!"
 
-[import.greeters]
+[pull.greeters]
```

Comment:
`import` is more suitable. There is no pull happening at this step, we import deployed contracts (addresses) from chain.

Reply:
I generally agree with this, but I'm sympathetic to the point that `import` in the step and `imports` in the context trips people up.

Reply:
May I suggest to not rename anything that already exists and just focus on what was giving the most pain. After some extra reading, new names do not make more sense than old ones, but introducing such a big breaking change will add a lot of friction.